framework/src/org/checkerframework/framework/type/AnnotationClassLoader.java

```diff
@@ -683,40 +652,53 @@ public int compare(File o1, File o2) {
      * annotation that is supported by a checker.
      *
      * @param fullyQualifiedClassName the fully qualified name of the class
-     * @return the loaded annotation class if it is defined with ElementType.TYPE_USE and is a
-     *     supported annotation, null otherwise
+     * @param issueWarnings set to true to issue a warning when a loaded annotation is not a type
+     *     annotation. It is useful to set this to true if a given annotation must be a well-defined
+     *     type annotation (eg annotation class names given as command line arguments).
+     * @return the loaded annotation class if it has a {@code @Target} meta-annotation with the
+     *     required ElementType values, and is a supported annotation by a checker. If the
+     *     annotation is not supported by a checker, null is returned.
      */
-    private final /*@Nullable*/ Class<? extends Annotation> loadAnnotationClass(
-            final String fullyQualifiedClassName) {
-        Class<?> cls = null;
+    protected final /*@Nullable*/ Class<? extends Annotation> loadAnnotationClass(
+            final String fullyQualifiedClassName, boolean issueWarnings) {
 
         try {
```

Comment:
Limit the scope of try blocks, to only cover the lines that can actually throw that exception.

Reply:
Fixed